library/src/G4CMPCrystalGroup.cc

```diff
@@ -79,11 +80,9 @@ void G4CMPCrystalGroup::SetTriclinic(G4double al, G4double bt,
 G4bool G4CMPCrystalGroup::FillElReduced(G4double Cij[6][6]) const {
   switch (group) {
   case amorphous:    Cij[3][3] = 0.5*(Cij[0][0]-Cij[0][1]); // Cubic, C44 set
-                     [[fallthrough]];
```

Comment:
Why did you remove all of the `[[fallthrough]]` directives?  Those are needed to avoid a whole lot of compiler warnings.

Reply:
In particular, these are the code changes which Stefan referenced in the comment dated 20240426.  Did you restore the comment "by hand", without restoring the missing code?